### DISCUSSION TITLE
httpx error

### DISCUSSION THREAD

**Discussion by EntroVyx** [robotshell/magicRecon#11] [Q&A]
Do you know what it could be?

[+] Httpx 
Checking alive subdomains...

Usage: httpx [OPTIONS] URL

Error: No such option: -l
cp: cannot stat 'alive.txt': No such file or directory
sed: can't read alive_subdomains.txt: No such file or directory
sed: can't read alive_subdomains.txt: No such file or directory
sort: cannot read: alive_subdomains.txt: No such file or directory
cat: alive_subdomains.txt: No such file or directory
sed: can't read alive_subdomains.txt: No such file or directory
cat: alive_subdomains.txt: No such file or directory

**Comment by MoffysDev** [comment]
Likely the script uses -l flag which is not supported in newer httpx version. Need to check usage: httpx [OPTIONS] URL. The -l flag maybe was for input file? Let’s recall: ProjectDiscovery's httpx has flags: -l for list input file? I think httpx supports -l to read URLs from a file. But error says no such option: -l, maybe they are using an older version? Let’s check: httpx -help shows: Usage: httpx [options] -list FILE or -l FILE? The error indicates they passed -l but httpx doesn't recognize it. Perhaps they need to use -l flag correctly: httpx -l alive.txt? The error says no such option: -l, so maybe they need to use -l flag with a capital L? Let's quickly think: In httpx v1.x, the flag for input file is -l (lowercase L) for list? I recall that httpx uses -l for list, -o for output. But error says no such option: -l, maybe they are using an outdated version where the flag is -L? Another possibility: They need to install httpx via go install -v github.com/projectdiscovery/httpx/cmd/httpx@latest, but they have an older version.
